.github/workflows/ci.yml

```diff
@@ -14,28 +14,25 @@ jobs:
     name: Execute SmartUI Test with Github App Integration
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+      - uses: actions/checkout@v1
 
-    - name: Create commit status
-      run: |
-        API_HOST=https://api.github.com
-        # Check out the PR branch
-        git checkout $GITHUB_HEAD_REF
-        # Get the commit ID of the last commit
-        COMMIT_ID=$(git rev-parse HEAD)
-        echo "Commit ID of PR: $COMMIT_ID"
-        GITHUB_URL=$API_HOST/repos/$GITHUB_REPOSITORY/statuses/$COMMIT_ID
-        echo "GITHUB_URL: $GITHUB_URL"
-        echo "GITHUB_URL=$GITHUB_URL" >> $GITHUB_ENV
-   
-    - name: Install Dependencies
-      run: npm install
+      - name: Create commit status
+        run: |
+          API_HOST=https://api.github.com
+          # Check out the PR branch
+          git checkout $GITHUB_HEAD_REF
+          # Get the commit ID of the last commit
+          COMMIT_ID=$(git rev-parse HEAD)
+          echo "Commit ID of PR: $COMMIT_ID"
+          GITHUB_URL=$API_HOST/repos/$GITHUB_REPOSITORY/statuses/$COMMIT_ID
+          echo "GITHUB_URL: $GITHUB_URL"
+          echo "GITHUB_URL=$GITHUB_URL" >> $GITHUB_ENV
 
-    - name: Execute Test
-      run: npm test
+      - name: Install Dependencies
+        run: npm install
 
-    -name: Run all tests
-      run: npm run start
-    
-    -name: Check multiple URLS with LazyLoading
-      run: npm run multiple
\ No newline at end of file
+      - name: Execute Test
+        run: npm run start
```

Comment:
Please add command `npm test` in `Execute Test` section required for Github PR check

Reply:
Done